packages/wallets/src/components/WalletListHeader/WalletListHeader.tsx

```diff
@@ -48,7 +48,7 @@ const WalletListHeader: React.FC = () => {
 
     const handleToggle = () => {
         setIsChecked(prev => !prev);
-        if (isEuRegion && !hasAnyActiveRealWallets && isDemo) {
+        if (!hasAnyActiveRealWallets && isDemo) {
```

Comment:
Redirection to OC real account creation via real/demo switcher for demo only wallets
https://github.com/user-attachments/assets/01e9e94d-7b4a-48d2-8071-723cdc9088b0